### PR TITLE
Provide management command for checking HTML (refs #158)

### PR DESCRIPTION
**Pull request by Almad**
Adds `checkcreationshtml.py` commands. 

Some polution thanks to running `black` :)

### CODE REVIEW

File: ddcz/html.py [hunk 1]
```diff
@@ -250,3 +251,35 @@ def encode_valid_html(entity_string):
...
     return encoded_safe_string
+
+
+class HtmlChecker(HTMLParser):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.opened_tags = []
+
+    def handle_starttag(self, tag, attrs):
+        if tag not in ['br']:
```
Comment (jimmeak) [inline ddcz/html.py:262]: Why is only br in here? Don't we need also `hr` in here or `input`s?

File: ddcz/html.py [hunk 2]
```diff
@@ -250,3 +251,35 @@ def encode_valid_html(entity_string):
...
+
+        if self.opened_tags[-1] != tag:
+            raise HtmlTagMismatchException(
+                f"Unclosed tag {self.opened_tags[-1]} when tag {tag} encountered"
+            )
+        del self.opened_tags[-1]
+
+
+class HtmlTagMismatchException(Exception):
+    """ Pairing of the tags is not done properly """
```
Comment (jimmeak) [inline ddcz/html.py:279]: This will be my lack of knowledge of Python, but is this really correctly written?

Comment (Almad) [reply ddcz/html.py:279]: Yes, those are [docstrings](https://www.python.org/dev/peps/pep-0257/)

File: ddcz/html.py (hunk 1 above)
Comment (Almad) [reply ddcz/html.py:262]: `hr` is a good possibility, but should be checked. Inputs are definitely not allowed within article text.

**Review by jimmeak — APPROVED** [review]

**Pull request merged**